### PR TITLE
shutdown: add force option

### PR DESCRIPTION
**Pull request by d0gey**
Hello,

I've added 2 more examples to shutdown, teaching users to use /f flag to force-close all open apps.

Thank you

### CODE REVIEW

File: pages/windows/shutdown.md [hunk 1]
```diff
@@ -6,10 +6,18 @@
...
 
+- Shutdown the current machine force-closing all apps:
+
+`shutdown /s /f`
+
 - Restart the current machine:
 
 `shutdown /r`
 
+- Restart the current machine force-closing all apps:
```
Comment (mfrw) [inline pages/windows/shutdown.md]:
```suggestion

```

File: pages/windows/shutdown.md [hunk 2]
```diff
@@ -6,10 +6,18 @@
...
+
```
Comment (mfrw) [inline pages/windows/shutdown.md]:
```suggestion
```

File: pages/windows/shutdown.md [hunk 3]
```diff
@@ -6,10 +6,18 @@
...
+`shutdown /r /f`
```
Comment (mfrw) [inline pages/windows/shutdown.md]:
```suggestion
```

**Comment by sbrl** [comment]
Hey, @d0gey! I think what @mfrw is getting at here is that tldr pages can only have a maximum of 8 examples each. Are there some less commonly used examples we can remove here or refactor to bring us back under the limit?

**Comment by agnivade** [comment]
I did a rebase to preserve both of your changes.

**Pull request merged**